controller/users.go

```diff
@@ -117,6 +147,7 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
+		keycloakUserProfile.Update(tokenString, "http://sso.prod-preview.openshift.io/auth/realms/fabric8-test/account")
```

Comment:
@sbose78 Don't we have a function that gives us this URL without having it hardcoded ?